client/src/components/generic/TagsComponent/TagsComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react"
+import CloseSign from "assets/icons/close_sign.svg?react"
+
+type tagVariants = "primary" | "interactive"
+
+interface TagProps {
+  children: string
+  variant: tagVariants
+}
+
+type props = TagProps
+
+const PrimaryTagsComponent = ({ children }: props) => {
+  return (
+    <div className="bg-light-blue-60 flex h-[32px] w-[80px] items-center justify-center overflow-hidden whitespace-nowrap rounded-full">
+      <h5 className="font-bold">{children}</h5>
+    </div>
+  )
+}
+
+const InteractiveTagsComponent = ({ children }: props) => {
+  const [isVisible, setIsVisible] = useState(true)
+  const handleClose = () => {
+    setIsVisible(false)
```

Comment:
gj on this one, another note tho can we let consumers pass in a custom handler like `onClose` type `() => void` (i.e we might be using this to display filters, and when we close one of the tags we want to remove the filter)